virtual-setup/slack-procedures.md

```diff
@@ -1 +1,118 @@
-## Using Slack for CCDL virtual workshops
\ No newline at end of file
+# Using Slack for CCDL virtual workshops
+
+We use Slack for communication during CCDL virtual workshops.
+
+<p><img style = "padding: 0 15px; float: left;" src = https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/master/logo/slack-cancer-data-science-logo.png width = 50></p>
+<p style="margin-top: 20px;"> </p>
+<p>
+
+
+Specifically, we use the **Cancer Data Science Slack** team administered by the CCDL.
+You can join Cancer Data Science Slack by following this link: [ccdatalab.org/slack](https://www.ccdatalab.org/slack)
+<br>
+<br>
+
+All Slack communications must adhere to our [code of conduct](https://github.com/AlexsLemonade/2020-may-training/blob/master/code-of-conduct.md).
+
+#### Instructions for setting up Slack
+
+We have instructions for setting up Slack for your operation system available.
+
+* [Linux](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/linux-instructions.md#slack)
+* [Mac](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/mac-instructions.md#slack)
+* [Windows](https://github.com/AlexsLemonade/training-modules/blob/master/virtual-setup/windows-instructions.md#slack)
+
+#### Table of Contents  
+
+- [How we use Slack during workshops](#how-we-use-slack-during-workshops)
+  - [Using the training-specific channel](#using-the-training-specific-channel)
+  - [Using Slack calling during training](#using-slack-calling-during-training)
+  - [Using direct messages during training](#using-direct-messages-during-training)
+- [General Slack use](#general-slack-use)
+    - [Attaching a file or image](#attaching-a-file-or-image)
+    - [Adding code blocks to messages](#adding-code-blocks-to-messages)
+
+## How we use Slack during workshops
+
+Course instructors will add you to a private channel specific to your training a few days before the start of your training workshop.
+
+If you are not added to the training specific channel or are having trouble getting started with Slack, please [direct message](#using-direct-messages-during-training) a CCDL staff member (`Chante Bethell`, `Candace Savonen`, `Josh Shapiro` or `Jaclyn N. Taroni`) in Cancer Data Science Slack or email [training@ccdatalab.org](mailto:training@ccdatalab.org).
```

Comment:
Should we add a timeframe here? "If you are not added to the channel by X days before the start of training..."

Reply:
3 days?

Reply:
Sounds reasonable to me.